### PR TITLE
Add a message and a note about improved plotting by increased concurrency

### PR DESCRIPTION
**Pull request by EmilFattakhov**
This PR reflects changes introduced in the monorepo [PR 2536](https://github.com/subspace/subspace/pull/2536).
I've also added a short explanation to the cli-tips section, as some users will benefit from changing the default behavior.

### CODE REVIEW

**Comment by EmilFattakhov** [comment]
@jim-counter did you say that your CPU load also increased significantly? I can't test the new release myself (and won't be able to test for some time), but I was under the impression that CPU load should decrease, while RAM load increases. Happy to change it if that's not what is happening.

**Comment by jim-counter** [comment]
Talking specifically about Space Acres v0.1.3, the default behaviour of the farmer pins all cores on my CPU to 100% and it farms the fastest it ever has.

I think we mean that the `feb-19` build leverages the changes from the `feb-15` build and is now optimised to use resources to the maximum. This default behaviour can be over-ridden by setting `--record-encoding-concurrency 1`. CPU load increases because it is more efficient at being able to use every cycle. @nazar-pc is this accurate?

**Comment by nazar-pc** [comment]
> This default behaviour can be over-ridden by setting --record-encoding-concurrency 1. CPU load increases because it is more efficient at being able to use every cycle. @nazar-pc is this accurate?

This is accurate, but a small note on `This default behaviour can be over-ridden by setting --record-encoding-concurrency 1. CPU load increases because it is more efficient at being able to use every cycle. @nazar-pc is this accurate?`, it doesn't quite reach the same performance as `feb-05` because the PoSpace is still optimized for throughput, not latency. I am thinking whether it is worth adding some tuning for this option to Space Acres to allow users to choose how much load they want during plotting. Not sure what the selection should be though.

**Comment by jim-counter** [comment]
> I am thinking whether it is worth adding some tuning for this option to Space Acres to allow users to choose how much load they want during plotting. Not sure what the selection should be though.

_I_ would be grateful for this option. Selecting target CPU load as a percentage would be perfect but I have no idea if that's even possible. Any parameter to dial would work really - it's just how intuitive it would be. Happy to beta test any early builds if you want to experiment.

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 1]
```diff
@@ -158,6 +170,12 @@ Example:
 
 ### Optimizing DSN Syncing
 
+:::note
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx:173]: :(

**Review by jim-counter — APPROVED** [review]
LGTM!

**Comment by EmilFattakhov** [comment]
> I am thinking whether it is worth adding some tuning for this option to Space Acres to allow users to choose how much load they want during plotting. Not sure what the selection should be though.

+1 on this, would be great to be able to adjust the load, in case you need to keep the PC somewhat usable at the same time.

**Pull request merged**